src/internal/m365/exchange/restore.go

```diff
@@ -40,16 +40,13 @@ func ConsumeRestoreCollections(
 	}
 
 	var (
-		userID         = dcs[0].FullPath().ResourceOwner()
+		userID         = rcc.ProtectedResource.ID()
```

Comment:
nit: Call it `protectedResourceID` instead of `userID`?

Reply:
I'll go with `resourceID` for a middle ground.